params/config.go

```diff
@@ -590,6 +603,10 @@ func (c *ChainConfig) IsIstanbul(num *big.Int) bool {
 
 // IsBerlin returns whether num is either equal to the Berlin fork block or greater.
 func (c *ChainConfig) IsBerlin(num *big.Int) bool {
+	// Check if it is Hertz, which works on BSC
```

Comment:
To reviewer: please help to check whether the change here is ok, the purpose is to check that IsBerlin features are included for BSC as it assumed the Hertz to cover all Berlin features.

Reply:
ok for me

Reply:
I think this might not be ok after all. In `core/types/transaction_signing.go` the function `LatestSigner(config *params.ChainConfig)` checks for `config.BerlinBlock` to determine the Signer.